requirements.txt

```diff
@@ -1,3 +1,2 @@
-awscli>=1.16.195
```

Comment:
Removed this unneeded dependency because it was causing my local environment issues.